mantidproject/mantid
# Reflectometry interface (polref): Prevent progress bar activity if confirmation fails

As described in the related issue #17690, if 'process rows' is clicked without selecting any rows a confirmation dialog popup appears. If 'No' is chosen, no runs will be processed, but the progress bar shows activity as though processing was occurring. This PR fixes this issue.

**To test:**

- Access the interface from Interfaces->Reflectometry->ISIS Reflectometry (Polref)
- Click on the 'Process' button without selecting runs (loading runs isn't necessary here anyway). The confirmation dialog should come up. Click 'No' on this.
- Ensure the progress bar on the bottom does not show any activity after doing this.

Fixes #17690.

*Release notes have been updated.*

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
